clinica/pydra/pet_volume/pet_volume_cli.py

```diff
@@ -0,0 +1,115 @@
+from typing import List, Optional, Tuple
+
+import click
+
+import clinica.pydra.engine_utils as pydra_utils
+import clinica.pydra.pet_volume.pipeline as pydra_pet_volume
+from clinica.pipelines import cli_param
+
+pipeline_name = "pydra-pet-volume"
+
+
+@click.command(name=pipeline_name, hidden=True)
+@cli_param.argument.bids_directory
+@cli_param.argument.caps_directory
+@cli_param.argument.group_label
+@cli_param.argument.acq_label
+@cli_param.argument.suvr_reference_region
+@cli_param.option_group.pipeline_specific_options
+@cli_param.option.pvc_psf_tsv
+@cli_param.option_group.common_pipelines_options
+@cli_param.option.subjects_sessions_tsv
+@cli_param.option.working_directory
+@cli_param.option.n_procs
+@cli_param.option.yes
+@cli_param.option_group.advanced_pipeline_options
+@cli_param.option_group.option(
+    "-mask",
+    "--mask_tissues",
+    multiple=True,
+    type=click.IntRange(1, 7),
+    default=[1, 2, 3],
+    help=(
+        "Tissue classes (1: gray matter (GM), 2: white matter (WM), 3: cerebrospinal "
+        "fluid (CSF), 4: bone, 5: soft-tissue, 6: background) to use "
+        "for masking the PET image (default: GM, WM and CSF i.e. --mask_tissues 1 2 3)."
+    ),
+)
+@cli_param.option_group.option(
+    "-threshold",
+    "--mask_threshold",
+    default=0.3,
+    show_default=True,
+    help="Value used as threshold to binarize the tissue maps.",
+)
+@cli_param.option_group.option(
+    "-pvc_mask",
+    "--pvc_mask_tissues",
+    multiple=True,
+    type=click.IntRange(1, 7),
+    default=[1, 2, 3],
+    help=(
+        "Tissue classes (1: gray matter (GM), 2: white matter (WM), 3: cerebrospinal "
+        "fluid (CSF), 4: bone, 5: soft-tissue, 6: background) to use "
+        "as mask for PVC (default: GM, WM and CSF i.e. --pvc_mask_tissues 1 2 3)."
+    ),
+)
+@cli_param.option.smooth
+def cli(
+    bids_directory: str,
+    caps_directory: str,
+    group_label: str,
+    acq_label: str,
+    suvr_reference_region: Optional[str] = None,
+    pvc_psf_tsv: Optional[str] = None,
+    mask_tissues: List[int] = (1, 2, 3),
+    mask_threshold: float = 0.3,
+    pvc_mask_tissues: List[int] = (1, 2, 3),
+    smooth: List[float] = [8.0],
```

Comment:
On the nipype version of the pipeline, the value of `smooth` is `List[int]= (8, 8)`. Is this change right. TBH, i do not know what it does

Reply:
And if it is a single value, is it normal to use a list ?